pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -355,50 +418,78 @@ def make_package(args):
                                   key=LooseVersion)
     build_tools_version = build_tools_versions[-1]
 
+    # Folder name for launcher (used by SDL2 bootstrap)
+    url_scheme = 'kivy'
+
+    # Render out android manifest:
+    manifest_path = "src/main/AndroidManifest.xml"
+    if get_bootstrap_name() != "sdl2":
+        manifest_path = "AndroidManifest.xml"
+    render_args = {
+        "args": args,
+        "service": service,
+        "service_names": service_names,
+        "android_api": android_api
+    }
+    if get_bootstrap_name() == "sdl2":
+        render_args["url_scheme"] = url_scheme
     render(
         'AndroidManifest.tmpl.xml',
-        'src/main/AndroidManifest.xml',
-        args=args,
-        service=service,
-        service_names=service_names,
-        android_api=android_api,
-        url_scheme=url_scheme)
+        manifest_path,
+        **render_args)
 
     # Copy the AndroidManifest.xml to the dist root dir so that ant
     # can also use it
-    if exists('AndroidManifest.xml'):
-        remove('AndroidManifest.xml')
-    shutil.copy(join('src', 'main', 'AndroidManifest.xml'),
-                'AndroidManifest.xml')
-
-    render(
-        'strings.tmpl.xml',
-        'src/main/res/values/strings.xml',
-        args=args,
-        url_scheme=url_scheme,
-        private_version=str(time.time()))
+    if get_bootstrap_name() == "sdl2":
+        if exists('AndroidManifest.xml'):
+            remove('AndroidManifest.xml')
```

Comment:
Minor/question: won't the line below already override the existing file?

Reply:
Not for sdl2, because there manifest_path points to `src/main/AndroidManifest.xml`. This is also why only for sdl2, this copy line is meaningful. These subtle folder differences are also something that would make a lot of sense to get rid of, once I looked deeper at this - it's just a historical thing again that I just kept as-is, for now